scripts/release/packager/create-turf-xcframework.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function step { >&2 echo -e "\033[1m\033[36m* $@\033[0m"; }
+function finish { >&2 echo -en "\033[0m"; }
+trap finish EXIT
+
+GIT_TAG=${1}
+ARTIFACTS_DIRECTORY=${2}
+
+cd ${ARTIFACTS_DIRECTORY}
+
+step "Clone turf-swift"
+git clone https://github.com/mapbox/turf-swift.git
+cd turf-swift
+
+step "Checkout tag: ${GIT_TAG}"
+git checkout ${GIT_TAG}
+
+step "Run scripts/create-xcframework.sh"
+sh scripts/create-xcframework.sh
```

Comment:
This script doesn't exist in Turf 1.2.0, and there are no plans to merge it in. I think now might be a good opportunity to use this script as a starting point and generalize it sufficiently so that we can use it to build any of the XCFrameworks that we need to build from source.